go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -98,17 +98,17 @@ func (n *NodeController) AddPod(pod *kapi.Pod) error {
 		return n.DeletePod(pod)
 	}
 
-	externalGw := pod.Annotations[hotypes.HybridOverlayExternalGw]
+	externalGw, ok := pod.Annotations[hotypes.HybridOverlayExternalGw]
 	// validate the external gateway is a valid IP address
```

Comment:
@JacobTanenbaum how about "validate the external gateway (if any) is a valid IP address"

Reply:
done